docker/docker-compose.2004.54.yaml

```diff
@@ -21,8 +21,8 @@ services:
       - MAX_ALLOCS_ALLOWED_1000_addHandlers=47050
       - MAX_ALLOCS_ALLOWED_1000_addHandlers_sync=40050
       - MAX_ALLOCS_ALLOWED_1000_addRemoveHandlers_handlercontext=9035
-      - MAX_ALLOCS_ALLOWED_1000_addRemoveHandlers_handlername=18035  # improvement from 5.3 which was 19035
-      - MAX_ALLOCS_ALLOWED_1000_addRemoveHandlers_handlertype=18035  # improvement from 5.3 which was 19035
+      - MAX_ALLOCS_ALLOWED_1000_addRemoveHandlers_handlername=9035  # improvement from 5.3 which was 19035
```

Comment:
@Lukasa this comment isn't right anymore, 5.3 seems to be the same now?

Reply:
Yup, fixed.